cmd/server.go

```diff
@@ -57,6 +57,7 @@ const (
 	BitbucketUserFlag                = "bitbucket-user"
 	BitbucketWebhookSecretFlag       = "bitbucket-webhook-secret"
 	ConfigFlag                       = "config"
+	CheckoutDepthFlag                = "checkout-depth"
 	CheckoutStrategyFlag             = "checkout-strategy"
```

Comment:
can we move these two lines for `config-flag` in the sense of alphabetical order.

Reply:
@chenrui333 addressed feedback, please rereview when you get a chance